Penteract/Assets/Scripts/Fang.h

```diff
@@ -32,7 +31,7 @@ class Fang : public Player {
 	float dashCooldown = 5.f;
 	float dashSpeed = 100.f;
 	float dashDuration = 0.1f;
-
+	float trailDuration = 0.2f;
```

Comment:
This variable is not exposed in the player controller. If you want that somebody change this, you need to expose it. Otherwise, set it as private